pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -542,6 +542,15 @@ func (p *MasqueradePodInterface) createNatRules() error {
 		return err
 	}
 
+	if len(p.iface.Ports) == 0 {
```

Comment:
Why can't we just always do that? Pods also always forward every traffic, no matter if pods are specified or not.

Reply:
we can't do it because it will break istio intregration we can't forward everything into the vm if istio is running and the istio-proxy container is injected into the virt-launcher pod.

This is why I am going to add a PR into kubeivrt user-guide to explain that if the user use istio it **must** configure the ports

Reply:
Ok, can't say  that I like behaving different  by default to fit into the case where istio might be there or not. A unit test is missing. Apart from that good.

Reply:
Thanks I will work on the unit test for the case